spec/features/idv/steps/confirmation_step_spec.rb

```diff
@@ -16,10 +17,14 @@
       complete_idv_steps_before_confirmation_step(@user)
     end
 
-    it 'allows the user to refresh and still displays the personal key' do
+    it 'allows the user to refresh and still displays the same personal key' do
+      key_before_reload = scrape_personal_key
```

Comment:
I might have to revert these changes, since it appears we generate a new personal key every time the page reloads, since we assign (but don't save) the profile key. This existed previously as well. Alternatively, maybe we should be saving the profile after assigning the key?

https://github.com/18F/identity-idp/blob/2bd9626cf8ca8d45d688369eb71038deb9d9fc72/app/controllers/idv/personal_key_controller.rb#L70-L73

https://github.com/18F/identity-idp/blob/2bd9626cf8ca8d45d688369eb71038deb9d9fc72/app/models/profile.rb#L61-L68

Reply:
Reverted in fdbc75a61dfa313c5728596deb27f48740d6e290.